code/mob/living/critter/voidhound.dm

```diff
@@ -0,0 +1,61 @@
+//Quick, stealthy, squishy assassin
+//Take the heat off of the wraith by attracting attention
+
+/mob/living/critter/voidhound
```

Comment:
would appreciate a wraith abstract parent

Reply:
Done. Plaguerat, voidhound, spiker, nascent and skeleton commander now all use the wraith abstract parent.